server/@types/shared/models/Booking.ts

```diff
@@ -0,0 +1,20 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { Arrival } from './Arrival';
+import type { BookingBody } from './BookingBody';
+import type { Cancellation } from './Cancellation';
+import type { Departure } from './Departure';
+import type { Extension } from './Extension';
+import type { Nonarrival } from './Nonarrival';
+
+export type Booking = (BookingBody & {
+    status: 'arrived' | 'awaiting-arrival' | 'not-arrived' | 'departed' | 'cancelled';
```

Comment:
I'm really chuffed it generates union types from enums 👍

Reply:
Yeah it was an option for the package thankfully 😌